tests/lim/log_test.go

```diff
@@ -0,0 +1,129 @@
+package lim
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/lf-edge/eden/pkg/controller/elog"
+	tk "github.com/lf-edge/eden/pkg/evetestkit"
+	"github.com/lf-edge/eden/pkg/utils"
+)
+
+var eveNode *tk.EveNode
+var logT *testing.T
+
+const (
+	projectName  = "lim"
+	logToLookFor = "Disconnected"
+	logTimeout   = 30 * time.Second // 30 sec should be enough since it only takes 10 sec to upload logs with fast upload
```

Comment:
Are we certain the runners (which are VMs) can handle it consistently in 30 seconds? It's better than 10 seconds used previously, but nevertheless...

Reply:
the main issue is like you mentioned not getting the logs, but getting an SSH connection to EVE. this is the flaky part and I sadly still have no idea where it comes from. So if the ssh connection is not established during 30 seconds, the test will fail. 30 sec is definitely enough to get the logs